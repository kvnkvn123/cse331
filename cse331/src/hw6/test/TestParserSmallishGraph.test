# Test the parser on a graph that contains several nodes

LoadGraph sample1 sampleGraph1.tsv
ListNodes sample1
ListChildren sample1 aaa
ListChildren sample1 bbb
ListChildren sample1 ccc
ListChildren sample1 ddd