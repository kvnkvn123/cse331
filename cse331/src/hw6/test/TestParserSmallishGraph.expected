# Test the parser on a graph that contains several nodes

loaded graph sample1
sample1 contains: aaa bbb ccc ddd
the children of aaa in sample1 are: bbb(111) bbb(222) ccc(111)
the children of bbb in sample1 are: aaa(111) aaa(222) ccc(111)
the children of ccc in sample1 are: aaa(111) bbb(111) ddd(333)
the children of ddd in sample1 are: ccc(333)

sample1 already exists
sample1 already exists
garbage.tsv malformed